.circleci/config.yml

```diff
@@ -1,131 +1,138 @@
-defaults: &defaults
-  machine: true
+version: 2.1
+
+env: &env
   environment:
-    GRUNTWORK_INSTALLER_VERSION: v0.0.21
-    TERRATEST_LOG_PARSER_VERSION: v0.13.13
-    KUBERGRUNT_VERSION: v0.3.9
-    HELM_VERSION: v2.11.0
-    MODULE_CI_VERSION: v0.13.3
-    TERRAFORM_VERSION: 0.12.7
+    GRUNTWORK_INSTALLER_VERSION: v0.0.31
+    TERRATEST_LOG_PARSER_VERSION: v0.30.4
+    MODULE_CI_VERSION: v0.29.2
+    TERRAFORM_VERSION: 0.13.5
     TERRAGRUNT_VERSION: NONE
     PACKER_VERSION: NONE
-    GOLANG_VERSION: 1.11.2
+    GOLANG_VERSION: 1.14
     K8S_VERSION: v1.10.0
     KUBECONFIG: /home/circleci/.kube/config
 
+defaults: &defaults
+  machine: true
+  <<: *env
+
 install_gruntwork_utils: &install_gruntwork_utils
   name: install gruntwork utils
   command: |
     curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/master/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version "${GRUNTWORK_INSTALLER_VERSION}"
     gruntwork-install --module-name "gruntwork-module-circleci-helpers" --repo "https://github.com/gruntwork-io/module-ci" --tag "${MODULE_CI_VERSION}"
     gruntwork-install --binary-name "terratest_log_parser" --repo "https://github.com/gruntwork-io/terratest" --tag "${TERRATEST_LOG_PARSER_VERSION}"
+    gruntwork-install --module-name "kubernetes-circleci-helpers" --repo "https://github.com/gruntwork-io/module-ci" --tag "${MODULE_CI_VERSION}"
+    gruntwork-install --module-name "gcp-helpers" --repo "https://github.com/gruntwork-io/terraform-google-ci" --branch "gcp-helpers"
```

Comment:
NIT: Add a TODO to switch to the release version here. Got caught by this in https://github.com/gruntwork-io/terraform-aws-ci/pull/269.

Reply:
ok sure.